connectors/src/connectors/intercom/temporal/activities.ts

```diff
@@ -481,6 +501,20 @@ export async function syncTeamOnlyActivity({
     name: teamOnIntercom.name,
     lastUpsertedTs: new Date(currentSyncMs),
   });
+
+  // Also make sure a datasource folder node is created for the team
+  if (connector !== null) {
+    // Create folder node for the team
+    const teamInternalId = getTeamInternalId(connectorId, teamOnIntercom.id);
```

Comment:
TBH I'd rather have you use `teamOnDb` to have 1 way of doing this (same as when we get ContentNode)
if we think in Resources here, we would've had a way to get the parents, name and internalID from the Resource